jdaviz/tests/test_subsets.py

```diff
@@ -67,12 +69,12 @@ def test_region_from_subset_profile(jdaviz_app, spectral_cube_wcs):
     reg = subsets.get('Subset 1')
 
     assert len(subsets) == 1
-    assert isinstance(reg, RectanglePixelRegion)
+    assert isinstance(reg, SpectralRegion)
 
-    assert_allclose(reg.center.x, 2.25)
-    assert_allclose(reg.center.y, 128)
-    assert_allclose(reg.width, 2.5)
-    assert_allclose(reg.height, 256)
+    print(reg)
+
+    assert reg.lower == 4.0*u.Hz
+    assert reg.upper == 255.0 * u.Hz
```

Comment:
You can import this check as `from astropy.tests.helper import assert_quantity_allclose`.

```suggestion
    assert_quantity_allclose(reg.lower, 4.0 * u.Hz)
    assert_quantity_allclose(reg.upper, 255.0 * u.Hz)
```

In general, exact comparison is discouraged for floats because there can be floating point differences that we don't care about across different architectures.